src/day1/array_search0.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+int main()
+{
+	 int Ar[5],i,a,count=0;
+	 for(i=0;i<=4;i++)
+	 {
+		 printf("\n Enter the %d Number : ",i+1);
+		 scanf("%d",&Ar[i]);
+	 }
+	 printf("Enter the number you want to search");
+	 scanf("%d",&a);
+	 for(i=0;i<=4;i++)
+	 {
+		 if(Ar[i]==a)
+		 {
+			 count++;
```

Comment:
If the number is found at 1 position, why are you traversing entire array, Once the number has been found break the loop. Think about if you're searching an element in an array which contains one million elements, wouldn't it be a very bad program to search an entire array even if you have found the element within first few elements?